### DISCUSSION TITLE
Team PVP

### DISCUSSION THREAD

**Discussion by Sachmo1977** [chromiecraft/chromiecraft#3521] [Suggestions]
Hello, my name in game is Xane. I am currently making this post because I am deciding to quit playing the game. The main reason for me is that there is not any group PVP so I can play with my friends instead of against them. The game to me has kind of gotten stagnant, and PVP is a big part for me during this part of the slowdown in things to do. I love older WoW so much because of the social aspect, and when I cannot play PVP with my friends it is not fun to me at all. If there is any way that this could be addressed, I would most definitely come back and play, but for now I am going to do something else. Wish you all the best of luck going forward.

**Comment by Annamaria-CC** [comment]
Hey,

PvP group queue is disabled because it is causing server instability. We are working on fixing it.

Take care
